modules/node_modules/@ncigdc/utils/formatFileSize.js

```diff
@@ -1,31 +1,4 @@
 // @flow
+import filesize from 'filesize';
```

Comment:
This isn't a theoretically conversation - this site is in production already. They way it worked was the way we wanted it.